// @polkadot/api/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "Keyring", {
  enumerable: true,
  get: function () {
    return _keyring.Keyring;
  }
});
Object.defineProperty(exports, "WsProvider", {
  enumerable: true,
  get: function () {
    return _ws.WsProvider;
  }
});
Object.defineProperty(exports, "packageInfo", {
  enumerable: true,
  get: function () {
    return _packageInfo.packageInfo;
  }
});
Object.defineProperty(exports, "ApiPromise", {
  enumerable: true,
  get: function () {
    return _promise.ApiPromise;
  }
});
Object.defineProperty(exports, "SubmittableResult", {
  enumerable: true,
  get: function () {
    return _submittable.SubmittableResult;
  }
});
Object.defineProperty(exports, "ApiRx", {
  enumerable: true,
  get: function () {
    return _rx.ApiRx;
  }
});

require("./detectPackage");

var _keyring = require("@polkadot/keyring");

var _ws = require("@polkadot/rpc-provider/ws");

var _packageInfo = require("./packageInfo");

var _promise = require("./promise");

var _submittable = require("./submittable");

var _rx = require("./rx");
